bin/uno

```diff
@@ -107,6 +99,8 @@ version)
   echo "                         Run 'uno fetch' for a list of possible components."
   echo "  setup <component>      Sets up component and its dependencies (clearing any existing data)"
   echo "                         Run 'uno setup' for list of components."
+  echo "  start                 Start Zookeeper, Hadoop, Accumulo, if not running."
```

Comment:
Should be `start <component>     Starts component and it dependencies if not running`